<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PantallaInicio</class>
 <widget class="QDialog" name="PantallaInicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PrograPop</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>981</width>
     <height>654</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_nombre_usuario">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;Nombre Usuario:&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_puntaje">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;Puntaje&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>179</green>
              <blue>57</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>179</green>
              <blue>57</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Cooper Black</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>PROGRA</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Cooper Black</family>
          <pointsize>72</pointsize>
         </font>
        </property>
        <property name="text">
         <string>POP</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layout_salas_normales">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>SALAS NORMALES</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_s1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_s1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_s2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_s2">
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_s3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_s3">
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_s4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_s4">
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_s5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_s5">
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_s6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_s6">
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>SALAS ECUALIZADOR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_e1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_e1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_e2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_e2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_e3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_e3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_e4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_e4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_e5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_e5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_e6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_e6">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>SALAS 2x CANCION</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_21">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_22">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_23">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_24">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_25">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_26">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>label_11</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
